samples/bluetooth/connection_event_trigger/Kconfig.sysbuild

```diff
@@ -0,0 +1,10 @@
+#
+# Copyright (c) 2023 Nordic Semiconductor
```

Comment:
2024

Reply:
shall we just remove it? `Copyright (c) Nordic Semiconductor`

Reply:
idk what the guidelines are. I see only one file w/o in sdk-nrf so I think we should keep it.